crates/weaver_forge/README.md

```diff
@@ -201,6 +215,44 @@ e.g. \[\[a,b\],\[c\]\] => \[a,b,c\]
 - `attribute_namespace`: Converts {namespace}.{attribute_id} to {namespace}.
 - `required`: Filters a list of `Attribute`s to include only the required attributes. The "conditionally_required" attributes are not returned by this filter.
 - `not_required`: Filters a list of `Attribute`s to only include non-required attributes. The "conditionally_required" attributes are returned by this filter.
+- `markdown_to_html`: Converts a markdown string to an HTML string.
+- `text_map`: Converts an input into a string based on the `text_maps` section of the `weaver.yaml` configuration file and a named text_map.
+- `black`: Format a text using the black ansi code.
```

Comment:
Nit:  Since we only have a global namespace, can we prefix all of these with `ansi_*` to denote that they are for ANSI output?

Reply:
Ok. Sounds good.

Reply:
Done https://github.com/open-telemetry/weaver/pull/162/commits/459486ef859b6613915b9930965053c7474f5606